### DISCUSSION TITLE
isolation job

### DISCUSSION THREAD

**Discussion by ghost** [martin2250/OpenCNCPilot#191] [General]
Friends, scraping all the paths for insulation is very annoying and time consuming. I add a 0.1 mm thick path around the areas to be soldered. This way I save time. Does anyone have similar inventions other than solder mask?   :-)

![2](https://github.com/martin2250/OpenCNCPilot/assets/132232142/290bfdd4-2d02-49c4-80ca-9c73e72b1897)
![1](https://github.com/martin2250/OpenCNCPilot/assets/132232142/94f68e86-b636-4dad-9f0b-d13d66ad96e9)
![3](https://github.com/martin2250/OpenCNCPilot/assets/132232142/5c471ec1-88bb-4ea5-9acb-a2ee3557ca0a)

**Comment by martin2250** [comment]
not sure why the extra boxes around your pads end up in flatcam, maybe you imported an extra gerber layer? you should be fine with just the copper layer.

**Comment by ghost** [comment]
I don't want to dig around the roads. Processing time is getting longer.

![_20240504_133731](https://github.com/martin2250/OpenCNCPilot/assets/132232142/cdce935f-2f2b-4119-956c-b1573a326e48)

**Comment by deHarro** [comment]
Hi @mehmetcihanildiri

Very good idea!

First time I read your post over at issues, I didn't catch the point, but after Martin moved it to discussions, I read it again and got the idea.

You don't know an easy means of adding those boxes around to be soldered pads in EAGLE together with pcb-gcode.ulp automatically, do you?

Placing those spacings around 11 pads (your example) manually is feasible, but let's think bigger. Try that with 100 pads with different shapes (round, octagon, rectangular, oval, ...)

![RollerControl_4_board](https://github.com/user-attachments/assets/2188d697-5df8-4aa7-8d59-277a4d6b74fa)

No offense, just asking 😀

Harald
